Resources/Prototypes/Entities/Objects/Shields/shields.yml

```diff
@@ -66,7 +66,7 @@
   name: riot shield
   parent: [ BaseShield, BaseRestrictedContraband ]
   id: RiotShield
-  description: A large tower shield. Good for controlling crowds.
+  description: A shield built for protecting against melee attacks.
```

Comment:
Leave the original description. By default all shields do have the ability for cc (by anchoring). By specifying it protects against melee attacks it makes it sound like it doesn't work against anything else, which isn't true.